.github/workflows/python/integration_tests.yaml

```diff
@@ -0,0 +1,43 @@
+name: Integration tests
+
+on:
+  workflow_call:
+    secrets:
+      APIFY_TEST_USER_PYTHON_SDK_API_TOKEN:
+        description: API token of the Python SDK & client testing user on Apify
+        required: true
+
+# This is to make sure that only one run of this workflow is running at the same time,
+# to not overshoot the test user limits.
+concurrency:
+  group: integration_tests
+
+jobs:
+  integration_tests:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Run integration tests only on the oldest and newest supported Python versions,
+        # as these tests are time-consuming and these versions are the most likely to encounter issues.
+        python-version: ["3.9", "3.12"]
+      max-parallel: 1 # no concurrency on this level, to not overshoot the test user limits
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          pipx install --python ${{ matrix.python-version }} poetry
+          make install-dev
+
+      - name: Run integration tests
+        run: make INTEGRATION_TESTS_CONCURRENCY=8 integration-tests
+        env:
+          APIFY_TEST_USER_API_TOKEN: ${{ secrets.APIFY_TEST_USER_PYTHON_SDK_API_TOKEN }}
```

Comment:
This is probably *the* part where we want to be extra careful. Any chance we could just copy the approval logic from Python SDK?